### ISSUE TITLE
[Learners] Solve Activity 1 : Activity 1: Getting Started

### ISSUE THREAD

**Issue by Kevinpaulose05** [Crio-Bytes/Miscellaneous#59]
**MicroByte Title:** COVID-19 Visualization using Pandas and Matplotlib

**Improvement Proposal:** 
- Solve activity 1 by updating the [code](https://github.com/Crio-Bytes/Miscellaneous/blob/main/COVID-19%20Visualization%20using%20Pandas%20and%20Matplotlib/src/Activity_1.ipynb) properly, preferably first by using my approach. Then you may and I suggest should improvise. 

**Benefits of proposed improvements:** 
- Learners will learn how libraries are imported in python, how datasets are read and how they can be displayed using juypter notebooks.

**Comment by itsadityagupta** [comment]
Hey @Kevinpaulose05 ! Will I get points to solve this issue? If yes, please assign me this.

Thanks.

**Comment by Kevinpaulose05** [comment]
> Hey @Kevinpaulose05 ! Will I get points to solve this issue? If yes, please assign me this.
> 
> Thanks.

Of course. I can't assure hard numbers as points but your contributions will be counted at the end of the day in the leaderboard.

**Comment by itsadityagupta** [comment]
Hey @Kevinpaulose05 ! Shall I create a new file named **Activity 1 Solution** or add the solutions to the same file under the heading **Solution**?

**Comment by Kevinpaulose05** [comment]
> Hey @Kevinpaulose05 ! Shall I create a new file named **Activity 1 Solution** or add the solutions to the same file under the heading **Solution**?

I am editing my PR. There will be a folder named activity 1. Drop ur solution inside it labelled activity 1 solution 1 types.

**Comment by Kevinpaulose05** [comment]
This is an **open issue** and all solutions are welcome. Each solution will be treated as a unique contribution. 
First person to solve this was @Aditya-Gupta1 . Kudos!
